### ISSUE TITLE
Code Table Request - Create locality attribute "waterbody", with values that are aquatic names from IHO (WITHOUT containing associated shapes)

### ISSUE THREAD

**Issue by genevieve-anderegg** [ArctosDB/arctos#7374]
## Instructions

_This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions._

_Reviewing documentation before proceeding will result in a more enjoyable experience._

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

As discussed today in the Arctos Geography Committee: Collections with aquatic specimens need a way to both associate in the catalog record 1) the administrative area associated with the specimen for permitting purposes, and 2) the non-administrative name of the water body the specimen was collected in. Collections have aquatic specimens with varying levels of detail of collection data (a lot to very little), which means specimens with sparse collection data (e.g. "Pacific Ocean") are often not georeferenced. However, it would be ideal to be able to curatorially apply the names of water bodies to catalog record specimen collection events regardless if they are georeferenced or not. Currently, the ability to ingest new shape files into Arctos and the associated cleanup/association of various shape files in different spatial layers (which will be needed for specimens collected on "edges", like "off the coast of Florida, in the Atlantic Ocean") is limited by manpower, processing power, and funding. Therefore, the Geography Committee proposes that the names of water bodies should be able to be linked to catalog records through a controlled Locality Attribute table (with NO associated shape files attached to water body names) so that collections are able to more easily attach the name of a water body in a controlled name to a record (which will assist with searching within Arctos, allowing to search for both records with and without geolocation and coordinates), and also bypassing the current issues with spatial data (https://github.com/ArctosDB/arctos/issues/6521). If/when the issues with spatial data is overcome, these data could also assist with linking shape files back to records via this locality attribute. Additionally, this attribute could be pointed at the waterbody field in aggregators such as GBIF to assist with searching in those services.

Geography members, **please** edit/add/correct anything stated here!! @dustymc @happiah-madson @sharpphyl @ekrimmel @falco-rk 

### Context
_Describe why this new value is necessary and existing values are not._

This new locality attribute will allow collections to link the name of a water body to a catalog record regardless of the quality of the specimen collection data

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlocality_attribute_type#

### Proposed Value

Locality attribute of "waterbody", with pick list of controlled names that are already in Arctos, currently associated with IHO and Marine Regions shape files, but WITHOUT linking to these shape files. Dusty has the names of waterbodies in Arctos from these services already, but this issue proposes just using the names of these shapes as an attribute (and NOT the shapes).

### Proposed Definition

~~See above~~

~~Any significant accumulation of water on the surface of Earth or another planet. The term most often refers to oceans, seas, and lakes, but it may include smaller pools of water such as ponds. A body of water does not have to be still or contained; rivers, streams, and canals are also considered bodies of water.~~

* from @Jegelewicz  in https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2163250149
* nevermind see https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2166251993

~~A body of water.~~

* from @Jegelewicz https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2166901785, sentensed.
* nevermind I hate this too

A body of water. Acceptable values are listed in https://www.marineregions.org/ with "PlaceType" in (IHO Sea Area). Documentation URL must be a marineregions.org MRGID. Water Body Type must be a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.

* from @dustymc  in https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2168430672

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

Pick list

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

High

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
Here's some data to get this started. I was hoping to somehow use higher geog as a start to documentation, but it doesn't seem very helpful and I think that's just going to make messes. Assuming this proceeds (I don't see any technical problems with it) It's probably ideal to open an issue for each of these BUT that would be overwhelming and these aren't exactly functional terms, so maybe it makes sense to create the code table from a spreadsheet. If that works, these (or whatever of them you want to create as ctwaterbody) would need a definition added.

[temp_maybewaterbody.csv](https://github.com/ArctosDB/arctos/files/14172525/temp_maybewaterbody.csv)

@ArctosDB/arctos-code-table-administrators does that sound right/reasonable?

----------------------------


Mostly unrelated, I also tried to pull geography from some DMNS coordinates. That's no problem to do, but it's slow and I think probably dangerously expensive - which still seems like some sort of 'database is broken' problem to me, but it's not one that I can see or fix. Anyway, here's the few records I processed, maybe it'll be useful....


```


 locality_id |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          geogs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
-------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
    10017175 | Mexico|North America|Pacific Ocean, Gulf of California|Mexico, Baja California
     1151427 | NONE
    10000187 | NONE
    10001001 | NONE
    10007230 | NONE
       80277 | NONE
    10001465 | NONE
    10075367 | North America|United States|United States, Florida|United States, Florida, Miami-Dade County
    10097902 | Solomon Islands, Central|Solomon Islands, Malaita|South Pacific Ocean|Solomon Islands, Isabel|Solomon Islands|Pacific Ocean, Solomon Sea
    10099964 | South Atlantic Ocean|Brazil, Bahia|Brazil|South America
    10097462 | South Island|South Pacific Ocean|New Zealand, Canterbury|New Zealand
    10000673 | Tanzania, Tanga|Africa|Somalia, Shabeellaha Hoose|Somalia, Bay|Somalia, Gedo|Somalia, Jubbada Dhexe|Somalia, Jubbada Hoose|Uganda|Tanzania|Ethiopia|Kenya, Isiolo|Uganda, Kumi|Uganda, Pallisa|Uganda, Lake Victoria|Uganda, Lira|Uganda, Luwero|Uganda, Masaka|Uganda, Mayuge|Uganda, Mbale|Uganda, Moroto|Uganda, Mpigi|Uganda, Mukono|Uganda, Nakapiripirit|Uganda, Nakasongola|Uganda, Sironko|Uganda, Soroti|Uganda, Wakiso|Uganda, Pader|Uganda, Apac|Uganda, Bugiri|Uganda, Busia|Uganda, Kitgum|Uganda, Iganga|Uganda, Jinja|Uganda, Kaberamaido|Uganda, Kalangala|Uganda, Kamuli|Uganda, Kapchorwa|Uganda, Katakwi|Uganda, Kayunga|Uganda, Kotido|Kenya, Baringo|Somalia|Uganda, Kampala|Uganda, Tororo|Tanzania, Kagera|Kenya, Laikipia|Indian Ocean|Tanzania, Manyara|Kenya, Kwale|Kenya, Samburu|Kenya, Kiambu|Kenya, Kericho|Kenya, Meru|Kenya, Narok|Kenya, Nairobi|Ethiopia, Oromia|Kenya, Bomet|Kenya, Bungoma|Kenya, Busia|Kenya, Elgeyo-Marakwet|Kenya, Embu|Kenya, Garissa|Kenya, Tana River|Tanzania, Simiyu|Kenya, Kitui|Kenya, Lamu|Kenya, Kilifi|Kenya, Mombasa|Tanzania, Singida|Kenya|Tanzania, Mwanza|Tanzania, Kilimanjaro|Ethiopia, Somali|Ethiopia, Southern Nations, Nationalities|Kenya, Nakuru|Kenya, Kisumu|Kenya, Machakos|Kenya, Makueni|Tanzania, Mara|Kenya, Marsabit|Tanzania, Arusha|Tanzania, Dodoma|Kenya, Kajiado|Kenya, West Pokot|Kenya, Kisii|Kenya, Migori|Kenya, Murang'a|Kenya, Nandi|Kenya, Nyamira|Kenya, Nyandarua|Kenya, Nyeri|Kenya, Siaya|Kenya, Taita Taveta|Kenya, Tharaka-Nithi|Kenya, Trans Nzoia|Kenya, Turkana|Kenya, Uasin Gishu|Kenya, Vihiga|Kenya, Wajir|Kenya, Homa Bay|Kenya, Kakamega|Kenya, Kirinyaga|Kenya, Mandera
     1156333 | The Coastal Waters of Southeast Alaska and British Columbia|United States|North America|Mitkoff|Kupreanof|United States, Alaska|Petersburg Quad|United States, Alaska, City and Borough of Wrangell
    10095356 | The Coastal Waters of Southeast Alaska and British Columbia|United States, Washington, Clallam County|United States, Washington|United States|North Pacific Ocean
     1150164 | United States, California, Humboldt County|United States, California|United States|North Pacific Ocean
     1161545 | United States, California, San Diego County|Mexico, Baja California|United States, California|Coronado|Mexico|United States|North America|North Pacific Ocean
     1114126 | United States, California, San Diego County|United States, California, Orange County|Mexico, Baja California|United States, California|United States, California, Imperial County|Coronado|Mexico|United States|United States, California, Riverside County|North America|North Pacific Ocean
     1131674 | United States, Colorado|United States, Colorado, Grand County|United States|North America
    10091758 | United States, Oregon, Multnomah County|United States, Oregon|Government|United States, Oregon, Clackamas County|United States, Oregon, Washington County|United States, Washington|United States|United States, Washington, Clark County|North America|North Pacific Ocean
    10095240 | United States, Washington, Mason County|United States, Washington, Pierce County|United States, Washington, Kittitas County|United States, Washington, Lewis County|United States, Washington, Kitsap County|Fox|Anderson|Blake|Bainbridge|United States, Washington, Grays Harbor County|Canada, British Columbia|Portage|The Coastal Waters of Southeast Alaska and British Columbia|United States, Washington, Skagit County|United States, Washington, Chelan County|United States, Washington, Clallam County|United States, Washington, Yakima County|Sucia|United States, Washington, Snohomish County|United States, Washington|United States, Washington, King County|Blakely|Pender|Portland|San Juan|Cypress|Tumbo|Mayne|Protection|Patos|Sidney|United States|Guemes|Henry|Saturna|United States, Washington, Thurston County|Burrows|Lumni|Marrowstone|Orcas|North America|Vancouver|Vashon|Waldron|United States, Washington, Island County|Piers|Samuel Island|Shaw|United States, Washington, Whatcom County|Sinclair|McNeill|Speiden|Decatur|Canada|United States, Washington, Jefferson County|United States, Washington, San Juan County|Allan|Lopez|Oak Harbor|Salt Spring|Coal|Discovery|Hat|Harstine|James


```

**Comment by sharpphyl** [comment]
@dustymc 

> would need a definition added.

I'm not sure what a definition for each entity is.  For example, the first entry Adriatic Seas has the following data in higher geography: 


Mediterranean Sea, Adriatic Sea | Mediterranean Sea | Adriatic Sea | https://en.wikipedia.org/wiki/Adriatic_Sea | spatial data   source iho_world_seas::Adriatic Sea | TRUE | 10007368

Do you need something more than this?  I was hoping that the table used to limit/structure the locality attribute "water body" could be the same as the IHO seas in Continents/Oceans or a subsection of it, but will work on definitions if something else is needed.

As Genna said, we are not expecting shape files to be associated with these entries as that may cause a conflict with the shape file of the administrative unit (GADM) also assigned to the catalog record.  We do want the data in this locality attribute to be mapped to [DC:water body](http://rs.tdwg.org/dwc/terms/waterBody) for consolidators.

@falco-rk and Hannah at OGL may have different needs, but this should be some help to getting their data into Arctos.

> here's the few records I processed,

Not having a before and after, I'm not sure what changes you made.  Should I try to figure that out or do you not plan to pursue this approach any further?

Thanks again for working with us to find a way to capture this data more effectively.

**Comment by Jegelewicz** [comment]
Will implementation mean removal of the IHO Seas from higher geography?

**Comment by dustymc** [comment]
>removal of the IHO Seas from higher geography?

Not at all.

The "functional" alternative right now would be to assert two localities

* California (from GADM)
* Pacific Ocean (from IHO)

The "maybe needs https://github.com/ArctosDB/arctos/issues/5597" alternative would be to find a spatial data source that does whatever's desired, or to create a "merge layer" using bits of whatever sources might do whatever's desired, make that an authority, and then just using it for beachy-things in CA would include California and Pacific Ocean. I'm not sure if this pursuit has been exhausted, but it probably should be before we go somewhere else.

This just lets someone say "Pacific Ocean" alongside a "normal" locality assertion - and doesn't have enough data to complain when the coordinates and assertion are in Florida (and that should probably be made clear from any eventual definitions, there's clearly some confusion in how this stuff works).

I think the goal involves https://github.com/ArctosDB/arctos/issues/7348 (but ???, and ???? re: how that fits with existing mappings). 

Maybe related to the above, maybe a shortcoming in our model, It was mentioned a few times that we need waterbody, I never managed to understand a clear example of that.

> first entry Adriatic Seas 

I was getting a bunch of them - these terms are unique here but components of bigger/smaller things in the source. I can try to look at it again when I can find time, get you a geog dump, ???? - and maybe @Jegelewicz can help figure out what's useful here. (I'm not sure these need defined, they're not functional in any way that I can see, but that idea hurts my brain...)

> same as the IHO seas in Continents/Oceans 

Same terms, but a different structure is necessary. (And the data I dumped isn't just sea, maybe I got lost, let me know if you need something else.)

>not expecting shape files

To be clear: That's just not possible here, it's a different kind of data in a different structure. 

>before and after,

I didn't change anything, I just ran your coordinates against existing spatial data (in an effort to help us all understand what a spatial approach to this means). Eg https://arctos.database.museum/editLocality.cfm?locality_id=10095240 you say "United States, Washington" but the spatial data (your coordinates) claim it could be in a couple dozen other things.

I was actually listening to some comments in the discussion and trying to read between the lines of @ekrimmel 's [survey](https://docs.google.com/document/d/1x4PQ5-qF0kdKOM6HCjellVG4noBHkBYiMlINdFmR6u4/edit#heading=h.k66o86up6sqa) wondering if ya'll wouldn't be happier if **all** of "higher geography" was something with which you could do WHATEVER, and the spatial stuff something completely separate, not asserted, just coordinate-derived. I got the idea that not allowing "Amundson Sea, Kansas" was seen as some sort of fault or drawback of Arctos, and that for various reasons (permits? tradition? previous political situations?? IDK...) you might want to assert a near-infinite number of things that look strange from a spatial perspective. New Issue if any of that's anywhere near the mark...

**Comment by Jegelewicz** [comment]
> wondering if ya'll wouldn't be happier if all of "higher geography" was something with which you could do WHATEVER, and the spatial stuff something completely separate

I think this is sorta true, but a little more nuanced. IMO we are squishing together two things that maybe shouldn't be squished together.

Textual descriptions of locality (even when they MIGHT have associated footprints) are generally subject to the whims of humans and nature. Today California is one shape, in 100 years, some of it has succumbed to the Pacific Ocean, and it is another. 34.0549, -118.2426 along with a datum is the same spot on the planet, no matter what - right? Making attempts to match those with the textual description and ignoring the time component just makes messes.

Perhaps crazy idea - the only REAL localities are 34.0549, -118.2426 along with a datum, everything else is an assertion that requires a time component to interpret properly (the datum is actually the time component for the coordinates), without it, even that point on the planet is unknowable. I think the idea that locality can be divested of the time component is what causes most of our troubles. I also think that we want to be able to assert things, even if they end up looking crazy and there shouldn't be any reason we cannot do both, it just needs to be clear what is what.

Sorry for the off-topic rant. If someone wants to say 34.0549, -118.2426 has Waterbody of Pacific Ocean, that is their prerogative. I might rely on the words, but more likely, I would rely on the coordinates and either way I might be mislead as both were entered by fallible humans. I do think people are looking for ways to make sure they don't make up "geography" so a code table format to ensure that I don't make a nonexistent combination of state and county is useful. But in a way - isn't that just an attribute of 34.0549, -118.2426 as much as Waterbody?

**Comment by dustymc** [comment]
>someone wants to say 34.0549, -118.2426 has Waterbody of Pacific Ocean, that is their prerogative

That is (I think) what this Issue is about.

>I would rely on the coordinates

I definitely don't give assertions much clout, and would encourage anyone running analyses to do the same! And hopefully the other stuff doesn't just look like 'Arctos is broken' to the world....

>time component

That's probably OK as a string (assert whatever you want), impossible under a controlled vocabulary ("France" has covered most of the planet over time - at least according to the French, a point which must clearly also be included), but is just trivial under a spatial perspective (toss it at whatever GIS layer you want). So now I'm wondering if trying to control this at all (or add complexity in any way) isn't a mistake....

**Comment by dustymc** [comment]
>geographic data of a specimen is just "Pacific Ocean", and geolocating that doesn't do a whole lot

Hard disagree from me. Maybe adding explicit point-radius error would make messes (and loop in a lot of land), but just the bare coordinates (which can ONLY defensibly be interpreted as 'unknown error' and NEVER as 'infinitely precise') would do it, or just use the shape.

**Comment by genevieve-anderegg** [comment]
> Maybe adding explicit point-radius error would make messes (and loop in a lot of land)

Yeah either you drop a point in the middle with an error radius that includes a ton of land (islands, all kinds of shoreline), it really isn't ideal. Didn't you say that would be really taxing on Arctos? (could be misremembering from yesterday)

> or just use the shape.

Which would be ideal. 

I think this issue is a good stepping stone to getting there.

**Comment by Jegelewicz** [comment]
The only way I would get on board with this proposal is if the "watery" components of higher geography are removed and there is only one place to assert bodies of water. Allowing the same term in two places will only confuse and de-normalize.

Once again, I feel the lack of a definition for higher geography is coming to bite us. Because things in higher geography can overlap, there are places where selecting it is an impossible choice. If higher geography in Arctos were limited to political boundaries, then it would be clear that Pacific Ocean belongs elsewhere. If all we do is add an attribute - at least half of those doing data entry will never use it because they already selected the same term as part of higher geography. No matter how hard anyone tries to avoid this it will happen.

The things from IHO Seas that are currently in higher geography could be moved to geographic features.

**Comment by dustymc** [comment]
>Allowing the same term in two places 

Well maybe, but the data are about as different as they can be so I don't think anyone who's doing much analysis of searching with any sort of precision is going to have much trouble telling strings from shapes.

> higher geography in Arctos were limited to political boundaries

That's not the conclusion of the geo group - Arctos is limited only to "authorities" and can't/won't/doesn't care about the contents of those authorities. The only real question is what constitutes an authority, but I'll support most anything that's 'published' somewhere. 

> things in higher geography can overlap, there are places where selecting it is an impossible choice.

OK, I'd probably oppose any "authority" that didn't have some sort of structure, but so far that's not been an issue and I don't think this is actually a consideration. 

> If all we do is add an attribute - at least half of those doing data entry will never use it because they already selected the same term as part of higher geography. 

Yes, any sort of independent assertion is going to be arbitrarily used. (That's precisely what I was trying to avoid with the 'georeference everything and let Arctos figure this out' approach that was so soundly rejected a couple years back.)

>The things from IHO Seas that are currently in higher geography could be moved to geographic features.

No, they cannot. They're not the same kind of data, locality attributes cannot support what's in geography. We could add the **terms** - not the data - to https://arctos-test.tacc.utexas.edu/info/ctDocumentation.cfm?table=ctlocality_attribute_type#feature / https://arctos-test.tacc.utexas.edu/info/ctDocumentation.cfm?table=ctfeature instead of something new if that serves the purpose.

**Comment by Jegelewicz** [comment]
> No, they cannot.

Again we are running into assertable features and "fun flexible features? We need to call these something different.

**Comment by sharpphyl** [comment]
What is the difference between adding "water body" with IHO/Marine Region listed entries in the dropdown table vs. our current "drainage" which has a similar (and somewhat overlapping) dropdown table?  What requirements did the items in the dropdown meet for "drainage" except that a collection needed that entity to accurately describe the locality?  If the Arctic Ocean is a drainage, isn't it also a water body?  

<img width="413" alt="Screenshot 2024-02-07 at 7 20 12 AM" src="https://github.com/ArctosDB/arctos/assets/15368365/10b568dc-9197-4fbd-bb5a-a9a1d3b5bb11">

**Comment by campmlc** [comment]
When we originally requested drainage be added for our fish collection, the values we needed to migrate included both text terms (Rio Grande Drainage) and HUC terms for terrestrial drainages. https://water.usgs.gov/GIS/huc.html

**Comment by dustymc** [comment]
>HUC

https://github.com/ArctosDB/arctos/issues/5223 could use some resolvin'


> IHO table for the water body dropdown a similar process

Yes and no, I think.  @Jegelewicz [excellent point](https://github.com/ArctosDB/arctos/issues/7374#issuecomment-1930719980) about normalization applies. Unlike watershed, for which we could find no relevant spatial data, we clearly have these spatial data, and there's a special dirty word just for scattering like things around in different-but-not-really baskets.... 

I no longer have no idea if we're solving an administrative problem or making a huge mess of the data here, it's seemed like both from time to time.

**Comment by genevieve-anderegg** [comment]
Having a waterbody locality attribute that is just the terms, and not the associated shapes, of waterbodies will help our collections begin to record our data in the way we want to. If one day all our wishlist items are met (#5597, #6521), then these attributes could be used to apply the shapes/geography associated with them to our catalog records (and all the intersects and edge cases can be resolved with an updated model/increased spatial capabilities). I think we should either go forward with this, because it will help in the meantime while we wait for Arctos to have better spatial capabilities. 

> solving an administrative problem or making a huge mess of the data here

It helps our administrative problems and I think sets us up for future successes with geography data once improvements to Arctos are made

**Comment by dustymc** [comment]
Need source-based definition: "appears in list at url" is ideal. (Anything in https://www.marineregions.org/ ? Some subset of https://www.marineregions.org/ ?)

**Comment by genevieve-anderegg** [comment]
Definition
1) called "waterBody" 2) use https://www.marineregions.org/  as source for code table requests 3) must be spatially explicit (even though the shape will not be linked with these terms)

**Comment by dustymc** [comment]
What's 'must be spatially explicit' mean? (Can you maybe find an example that's OK/should be approved for the code table and one that's not?)

Also - should this be limited to water? If so, how? Eg https://www.marineregions.org/gazetteer.php?p=details&id=32570 exists and as far as I can tell refers to the port, not the water. (And it's a point - is that 'spatial' and, either way, is there some way to tell other than zooming ?)

I still don't care what the answers to any of those questions are, I'd just like to have some formula such that that when someone requests eg [Agios Efstratios](http://marineregions.org/mrgid/14773) (or whatever) be added, we can all understand what the answer will be. (That one does seem to be 'spatial,' but a void in a waterbody rather than a waterbody...)

https://www.marineregions.org/gazetteer.php?p=details&id=19219 is a waterbody but not (exactly) spatial.

https://www.marineregions.org/gazetteer.php?p=details&id=8501 is someone's weird division?? I think from a publication, https://www.researchgate.net/figure/Brattegard-Holthe-1997-divided-the-Norwegian-coast-in-26-different-sectors-to-map-the_fig1_257733145

https://www.marineregions.org/gazetteer.php?p=details&id=64250 is in water...

(I searched 'rat' and clicked the interesting stuff...)

I think we need to narrow this down, somehow?

**Comment by genevieve-anderegg** [comment]
> What's 'must be spatially explicit' mean?

The name that someone wishes to add to this table must correspond to a shape in Marine Regions (even though the shape will not itself be added) 

> I think we need to narrow this down, somehow?

Yes agreed. Could we narrow it down by the category of the Marine Regions "Placetype? category?

![image](https://github.com/ArctosDB/arctos/assets/83782023/8e391a33-c7b8-4ebd-bfc9-5ed1cfae8cb8)

They have many options (n=327): 
[Marine Regions Placetype Terms.xlsx](https://github.com/ArctosDB/arctos/files/14670275/Marine.Regions.Placetype.Terms.xlsx)

Maybe OGL and us at DMNS can go through and identify Placetype term categories that have place shapes we want for our list of terms? That should cut out a lot of the weird things in Marine Regions. Then we can say that if we want to add a name from another Placetype category it has to go to the committee for discussion? 

Also, when identifying the names we want, maybe should work backwards from the DWC term for waterbody so that all of the terms we want added on this proposed code table match the [DWC waterbody definition](https://ltc.tdwg.org/terms/#GeographicContext_waterBody) , so that once we can link shapes to records in the way we want, everything is easily convertable

**Comment by dustymc** [comment]
>go to the committee for discussion

I don't like that because it's arbitrary. If at all possible, it would be useful to have some sort of go/nogo test for these where we're just following some bigger-than-Arctos "standard" rather than discussing every term.

>match the [DWC waterbody definition](https://ltc.tdwg.org/terms/#GeographicContext_waterBody)

Then let's do that instead of this??

> appropriate vocabularies include: HydroLAKES (https://www.hydrosheds.org/page/hydrolakes), a database aiming to provide the shoreline polygons of all global lakes with a surface area of at least 10 ha, the 'water body' term in the OBO ontology (http://purl.obolibrary.org/obo/ENVO_00000063) and, for marine water bodies, IHO Sea Areas (http://www.vliz.be/en/imis?dasid=5444&doiid=323)

**Comment by genevieve-anderegg** [comment]
> I don't like that because it's arbitrary. If at all possible, it would be useful to have some sort of go/nogo test for these where we're just following some bigger-than-Arctos "standard" rather than discussing every term

Totally agree

> Then let's do that instead of this??

I think doing something that follows the definition would steer us in the right direction. The definition does say:
"_**_Suggestions for appropriate vocabularies include:_**_ HydroLAKES (https://www.hydrosheds.org/page/hydrolakes), a database aiming to provide the shoreline polygons of all global lakes with a surface area of at least 10 ha, the 'water body' term in the OBO ontology (http://purl.obolibrary.org/obo/ENVO_00000063) and, for marine water bodies, IHO Sea Areas (http://www.vliz.be/en/imis?dasid=5444&doiid=323)." 

Emphasis mine. So the DwC definition does not lay out a hard and fast rule, but rather suggestions for following datasets (which was our plan, yay!). So, what if we follow the lead of the definition and choose a few specific datasets to include? 
Suggestions that would be useful for us at DMNS (and not full of clutter): HydroLakes and IHO Sea Areas (from above), and EEZs. Maybe there are some specific lists (PlaceTypes) from Marine Regions that would be useful too? 

What do we think? Would we rather make this a very broad field? @Jegelewicz @mkoo @happiah-madson @ekrimmel @falco-rk

**Comment by Jegelewicz** [comment]
> choose a few specific datasets to include?

I think this sounds like a good idea. We don't have a single authority for geography, why should we expect to find a single one here? Let's just make our list, then any terms that can be found in those authorities can be added to this code table without the need for all the box checking (but we should still make Github issues so that we can keep track of things).

**Comment by Jegelewicz** [comment]
> HydroLAKES (https://www.hydrosheds.org/page/hydrolakes), a database aiming to provide the shoreline polygons of all global lakes with a surface area of at least 10 ha
> 
> marine water bodies, IHO Sea Areas (http://www.vliz.be/en/imis?dasid=5444&doiid=323).

These seem like good places to start. HydroLAKES is a data download so maybe we just add the whole thing?

> the 'water body' term in the OBO ontology (http://purl.obolibrary.org/obo/ENVO_00000063) 

This isn't a list of waterbodies....

**Comment by genevieve-anderegg** [comment]
From last geography meeting:
Refining the `waterbody` code table definition (referencing DarwinCore definition):
1) table is named "waterBody" 
2) requested values must refer to vocabularies in the following datasets that are associated with a waterbody shape file (while the shape file itself will not be included in the code table) 
3) values can be added from the following datasets: HydroLAKES (https://www.hydrosheds.org/page/hydrolakes), which contains shoreline polygons for lakes larger than 10 hectares; IHO Sea Areas (http://www.vliz.be/en/imis?dasid=5444&doiid=323) for marine water bodies.

Edits/thoughts welcome, barring calls for larger discussion (#7666)...

**Comment by dustymc** [comment]
>table is named "waterBody"

hu?

>IHO Sea Areas

I'm still not crazy about that, those are already in Arctos as assertable geography, https://arctos.database.museum/place.cfm?sch=geog&valid_catalog_term_fg=1&geog_remark=iho. Having two ways of saying the same thing should require some (very!) special justification. (Or someone can perhaps explain to me how it's not the same and I'm just lost??)

**Comment by genevieve-anderegg** [comment]
> hu?

"waterbody" would be the name of the locality attribute, sorry for the confusion !

>  two ways of saying the same thing

Yeah not ideal, but if you need to say your shell was collected in the water off of Maine (for permitting/legal purposes) as well as the North Atlantic, then you could use Maine for your asserted higher geography and put North Atlantic in an attribute (per this request). Might need to tighten up the definition so we don't end up creating justification for having all the geo data in two places. But there really isn't a great way to store both the administrative and non-administrative watery geo data right now for all the needs these data need to meet (#7660). Maybe this goes on the work group/issues agenda...?

**Comment by dustymc** [comment]
>permitting/legal purposes)

WHY not use permits for that?!

>collected in the water off of Maine

In that case I think this request is actually sort of backwards. Spatially (assuming decent georeferences and such) that would be not-Maine, you can just use eg https://arctos.database.museum/place.cfm?action=detail&geog_auth_rec_id=1143 to get at 'waterbody.' The thing which can't correctly be said spatially is (offshore near) "Maine."

(And I'm still not seeing much problem with this, just trying to get all the gotchas worked out, maybe find some future direction hiding somewhere....)

**Comment by Jegelewicz** [comment]
I feel like I've asked this before, but what is our definition of "higher geography"? As we are currently mixing political and watery stuff, I don't think we have a definition that means we don't have overlapping polygons and this seems to me to be the root of our issues in the discussion here.

**Comment by dustymc** [comment]
>what is our definition of "higher geography"?

What SHOULD it be: https://github.com/ArctosDB/arctos/discussions/7666 ?

>I don't think we have a definition

Sure we do: https://handbook.arctosdb.org/documentation/higher-geography.html

**Comment by genevieve-anderegg** [comment]
I'd like this to keep moving forward. The reason we want Waterbody is so that if someone on GBIF searches 'Gulf of Mexico' in the Darwin Core field "Water body", our specimens come up even if they haven't been geolocated (which is obviously not ideal, but is reality).  Additionally, having this attribute will be very helpful for searching within Arctos, when the higher geography field is limited (often because we also need to record administrative areas as well).
Should this go on the list for the issues meeting tomorrow? Would bringing in other community members help? Or is this getting tabled because other discussions need to happen? #7666 
In any case, Geography meets next week, but this has been hanging around since the beginning of the year.

**Comment by Jegelewicz** [comment]
> Sure we do: https://handbook.arctosdb.org/documentation/higher-geography.html

That is a list of what is in there. I am interested in the concept we are trying to capture.

**Comment by dustymc** [comment]
More Geog Committee discussion

* locality attribute still seems like a good idea
* name is still unclear ("waterbody" or "waterBody" or "water body" or ???)
* "appears in HydroLAKES (https://www.hydrosheds.org/page/hydrolakes) or IHO Sea Areas (http://www.vliz.be/en/imis?dasid=5444&doiid=323)" is still the definition
    * yes there's overlap with geography, whatever, try not to notice....
* DO NOT try to batch or bulk anything, add one term at a time and let collections add attributes as they see fit (but we can help, just ask!)

**Comment by genevieve-anderegg** [comment]
> name is still unclear ("waterbody" or "waterBody" or "water body" or ???)

per DarwinCore "waterBody" http://rs.tdwg.org/dwc/terms/waterBody

**Comment by dustymc** [comment]
>per DarwinCore

I have no preferences, but that will be a mapping and DOES NOT need to influence our local terminology. "Consistent with https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlocality_attribute_type#" is my usual request, but I don't think there's anything consistent in there so whatever.....

**Comment by dustymc** [comment]
I think this is going in a different direction.

1. Kill this issue, we don't need a "generic wet thing" attribute
2. Create more focused attributes ('sea' and/or 'ocean' and/or 'oceanic region' and/or 'lake' and/or 'birdbath' and/or WHATEVER is useful in/to the collections)
3. Use them as proposed for this (and sorta incidentally, fewer values generally equals better UI, so yay)
4. maybe spatial-magic some of them (eg magic-add 'Arctic Ocean' to things which map there) for discovery purposes, be careful to use a bot determiner (easy to ignore for eg reports) to differentiate from collection-asserted if we do this
5. maybe display them in some special way in various places (mixed in with geography? Not sure I'm very clear on this....)
6. map them (all of them, some of them, whatever) to DWC:waterBody (and/or whatever) in whatever way seems most useful (super easy to change, but https://github.com/ArctosDB/arctos/issues/7348 doesn't seem to be going very fast...)

@mkoo et al. am I on the right page yet?

**Comment by Jegelewicz** [comment]
> Create more focused attributes ('sea' and/or 'ocean' and/or 'oceanic region' and/or 'lake' and/or 'birdbath' and/or WHATEVER is useful in/to the collections)

Why not use the unified chronostrat model? We have one attribute = waterBody BUT the terms have metadata

term | description | type | salinity | Source | whatelse?
 -- | -- | -- | -- | -- | -- 
Lake Erie | One of the five Great Lakes in North America. | lake | fresh | https://en.wikipedia.org/wiki/Lake_Erie | say whatever here? 
RioGrande | A river in North America | river | fresh | https://en.wikipedia.org/wiki/Rio_Grande | blah blah 
Aransas Bay | A bay and estuary in the Gulf of Mexico | bay | salt | https://en.wikipedia.org/wiki/Aransas_Bay | 

ONE place to look for watery stuff.

**Comment by genevieve-anderegg** [comment]
> Kill this issue, we don't need a "generic wet thing" attribute
Create more focused attributes ('sea' and/or 'ocean' and/or 'oceanic region' and/or 'lake' and/or 'birdbath' and/or WHATEVER is useful in/to the collections)
Use them as proposed for this (and sorta incidentally, fewer values generally equals better UI, so yay)
maybe spatial-magic some of them (eg magic-add 'Arctic Ocean' to things which map there) for discovery purposes, be careful to use a bot determiner (easy to ignore for eg reports) to differentiate from collection-asserted if we do this
maybe display them in some special way in various places (mixed in with geography? Not sure I'm very clear on this....)
map them (all of them, some of them, whatever) to DWC:waterBody (and/or whatever) in whatever way seems most useful (super easy to change, but https://github.com/ArctosDB/arctos/issues/7348 doesn't seem to be going very fast...)

This looks good to me and is what we agreed on in the meeting! 

However, 

> Why not use the unified chronostrat model? We have one attribute = waterBody BUT the terms have metadata

I feel like this could also definitely work.
As long as you can do multiple instances of the attribute with different values, they are searchable, are displayed in a useful and elegant way on the catalog record, and map to DWC:waterBody, either of these solution formats seems good to me.

Are there upsides/downsides to either one that I'm not thinking of that would lead us in one direction vs. the other? @mkoo @dustymc @happiah-madson @sharpphyl @ekrimmel @falco-rk

**Comment by dustymc** [comment]
The UCM upside is that adding 'muddy puddle' (a "subcategory") is a matter of DATA, not STRUCTURE - anyone with the right powers types some stuff and the category exists, I don't need to build code tables and add logs and checks and constraints and .... - it's easier and more scalable.

The UCM downside - IDK, shall we find out? (Nothing very obvious, MAYBE slightly more difficult to "columnify" but probably just different, ??, I'm not very scared.)

**Comment by genevieve-anderegg** [comment]
> Why not use the unified chronostrat model? We have one attribute = waterBody BUT the terms have metadata
> The UCM downside - IDK, shall we find out? (Nothing very obvious, MAYBE slightly more difficult to "columnify" but probably just different, ??, I'm not very scared.)

Been sitting on this for a few weeks now, and with several upsides and no obvious downsides to the Unified Chronostrat Model, I say lets go with that. I like Teresa's columns suggestions:
<html>
<body>



term | description | type | salinity | Source
Lake Erie | One of the five Great Lakes in North America. | lake | fresh | https://en.wikipedia.org/wiki/Lake_Erie 
RioGrande | A river in North America | river | fresh | https://en.wikipedia.org/wiki/Rio_Grande 
Aransas Bay | A bay and estuary in the Gulf of Mexico | bay | salt | https://en.wikipedia.org/wiki/Aransas_Bay

<br class="Apple-interchange-newline">
</body>
</html>


This will allow us to maintain one table (eliminating discussions of messy "hierarchy" in waterbodies and having to keep up multiple tables), and give the values meaningful context. Once this table is created, it would be great to have them displayed on the catalog record page (another issue later).

As per discussion with @mkoo in the last meeting, shall we start with the oceans? https://www.vliz.be/en/imis?dasid=5444&doiid=323
I advocate we first put in IHO Sea Areas (https://www.marineregions.org/gazetteer.php?p=image&pic=119334) with the North and South Pacific and Atlantic oceans combined to just be Pacific and Atlantic (there are records where the location is just "Pacific")

term | description | type | salinity | Source 
Pacific Ocean | Earth's largest and deepest ocean. | ocean | salt | (https://en.wikipedia.org/wiki/Pacific_Ocean)
Atlantic Ocean | Eath's second largest ocean, between the Americas to the west, and Africa and Europe to the east | ocean  | salt  | (https://en.wikipedia.org/wiki/Atlantic_Ocean)
Indian Ocean | Earth's third largest ocean, bounded by Africa to the west, Asia to the north, and Australia to the east | ocean  | salt  | (https://en.wikipedia.org/wiki/Indian_Ocean) 
Antarctic/Southern Ocean | Earth's fourth largest, circling around Antarctica and below 60° S latitude | ocean  | salt | (https://en.wikipedia.org/wiki/Southern_Ocean)
Arctic Ocean | Earth's smallest and shallowest ocean, including the North Pole and extending north from 60°N parallel | ocean  | salt  | (https://en.wikipedia.org/wiki/Arctic_Ocean)
(or whatever)

What do we think? We can work on specifics of terms and shapes this coming month, but everyone please weigh in on this format for our locality attribute!!! @mkoo @dustymc @happiah-madson @sharpphyl @ekrimmel @falco-rk

**Comment by Jegelewicz** [comment]
No slashes!

I say we just call that Southern Ocean and maybe we need one more column

term | description | type | salinity | Source | search terms 
 -- | -- | -- | -- | -- | -- 
Pacific Ocean | Earth's largest and deepest ocean. | ocean | salt | (https://en.wikipedia.org/wiki/Pacific_Ocean) | 
Atlantic Ocean | Eath's second largest ocean, between the Americas to the west, and Africa and Europe to the east | ocean  | salt  | (https://en.wikipedia.org/wiki/Atlantic_Ocean) | 
Indian Ocean | Earth's third largest ocean, bounded by Africa to the west, Asia to the north, and Australia to the east | ocean  | salt  | (https://en.wikipedia.org/wiki/Indian_Ocean) | 
Southern Ocean | Earth's fourth largest, circling around Antarctica and below 60° S latitude | ocean  | salt | (https://en.wikipedia.org/wiki/Southern_Ocean) | Antarctic Ocean
Arctic Ocean | Earth's smallest and shallowest ocean, including the North Pole and extending north from 60°N parallel | ocean  | salt  | (https://en.wikipedia.org/wiki/Arctic_Ocean) |

**Comment by genevieve-anderegg** [comment]
@Jegelewicz Perfect!
These are great terms to move forward and begin this code table with.
@dustymc Is anything else needed to make this happen?

**Comment by dustymc** [comment]
>anything else needed

The ever-elusive final approval/acceptance/social stuff; techy-bits look fine to me.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This is looking good to me. Box checked.

**Comment by dustymc** [comment]
Geog committee: GO

**Comment by falco-rk** [comment]
Sorry i missed that meeting! what specifically do you need from us? just the list of our water bodies?

**Comment by Jegelewicz** [comment]
@falco-rk nothing from you - that will be on us to get started then have you review.

**Comment by dustymc** [comment]
There's not much of a definition here. I modified TDWG's for now:

>The name of the water body in which the location occurs.

But I think maybe that's explicitly wrong - you may be trying to ***avoid*** the "within", and certainly nothing about this will prevent you from referencing anything from anywhere. What should I use?

(And @mkoo this could definitely benefit from an accessible test environment.)

**Comment by Jegelewicz** [comment]
I have complained about this at TDWG, but using the term in the definition is just bad. How about

> Any significant accumulation of water on the surface of Earth or another planet. The term most often refers to oceans, seas, and lakes, but it may include smaller pools of water such as ponds. A body of water does not have to be still or contained; rivers, streams, and canals are also considered bodies of water. 

Adapted from Wikipedia - https://en.wikipedia.org/wiki/Body_of_water

**Comment by mkoo** [comment]
> Adapted from Wikipedia - https://en.wikipedia.org/wiki/Body_of_water

ugh no. that's a tortured definition if I ever saw one! I think ```waterbody``` in Arctos is the polygon intersections from our chosen authorities -- so here's IHO, EEZ's, etc for oceans, seas, lakes, Exclusive Economic Zones, as listed in these authorities.

If we have a locality attribute of ```protected area``` then we adopt the WDPA db perhaps and the USPAD. And so forth. In other words we should definitely cite our authority sources.

Anyway, status update in Test?

**Comment by dustymc** [comment]
>no

I'll need SOMETHING to use as a definition.

>status

I believe mostly functional, but I am sorely missing an accessible test environment - I've made some substantial and far-ranging changes (https://github.com/ArctosDB/arctos/issues/7856), more eyeballs than mine on them before this hits production would be nice.

**Comment by Jegelewicz** [comment]
>  think waterbody in Arctos is the polygon intersections from our chosen authorities -- so here's IHO, EEZ's, etc for oceans, seas, lakes, Exclusive Economic Zones, as listed in these authorities.

This issue specifically says "Create locality attribute "waterbody", with values that are aquatic names from IHO (WITHOUT containing associated shapes)"?

**Comment by Jegelewicz** [comment]
> definition

a body of water

**Comment by dustymc** [comment]
I've updated the definition (including in my migration path) again, but upon further reflection I hate everything that's been proposed. None of these are **functional**, and having solid, deterministic definitions has become increasingly important as we grow as a community.

I propose the following. This would make code table admins lives easy/deterministic, and we can update the definition (through another code table request) at any time.

A body of water. Acceptable values are listed in https://www.marineregions.org with "PlaceType" in (IHO Sea Area). Documentation URL must be a marineregions.org MRGID. Water Body Type must be a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.

@sharpphyl @genevieve-anderegg and anyone else who might use this:

1. Does that approach work?
2. Are there any preemptive additions/expansions?

**Comment by mkoo** [comment]
I think this is what I've been poorly articulating... **base values on sources**. We can add authorities as needed and with CT admin approval. Looking forward to having this implemented!

**Comment by dustymc** [comment]
I am reopening this for further discussion. The [proposed definition](https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2168430672)...


>A body of water. Acceptable values are listed in https://www.marineregions.org/ with "PlaceType" in (IHO Sea Area). Documentation URL must be a marineregions.org MRGID. Water Body Type must be a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.


.. with which this was implemented is not the direction today's Committee discussions (https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit) went.

It's been proposed to bring individual terms from other groups in, without blessing the entirety of the group. We like http://marineregions.org/mrgid/1902 (``Atlantic Ocean``, a ``General Sea Area``) but  not http://marineregions.org/mrgid/8665 (``Off North West Amphipods``, another ``General Sea Area``). Can this be done in such a way that a future user won't point to the former while demanding the latter? (Or if we're OK with the latter, then can we just say that now?)

Is there some definition to which this can be updated ***before*** someone uses it with unrealistic expectations?

I think this should probably block https://github.com/ArctosDB/arctos/issues/7906, we should not implement a concept without a clear definition. (FWIW I still don't particularly care what that definition is, I still think lots of things could be useful for various purposes, I just want to know what we're trying to do before this turns into a mess.)

It was mentioned that The Committee could approve or reject individual terms, and I do strongly feel that we should avoid that situation if there's any possible way we can. Whether some Authority is or is not appropriate for a particular use should not depend on the whims of whoever might show up on some particular day!

**Comment by Jegelewicz** [comment]
Suggest the definition should be

Controlled vocabulary for bodies of water available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

* The Body of water is listed in in https://www.marineregions.org/ with "PlaceType" in (IHO Sea Area)
* Documentation URL is a marineregions.org MRGID.
* Water Body Type is a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.
     
    If the body of water does not meet those criteria, the documentation should include a link to the resource that describes the term and the description should include a short description of the body of water. 

Additional criteria can be added if we want to add a "preferred" source for freshwater etc.

**Comment by dustymc** [comment]
I am looking for a definition which defines/shapes the concept.  I don't think that's unreasonable for any Authority! 

I think you're implying that this is entirely arbitrary, if I can convince someone to check the boxes then my birdbath is in fact a waterbody?! That's probably OK, those data can still do lots of things, but let's drop the pretenses of following some authority if that's where this is going.

>freshwater

Adding all IHO PlaceType 'Lake' (or some other lake-source or whatever) doesn't change the shape of the concept in the same way that adding some arbitrary one-off does.  I'm not necessarily suggesting we don't add those one-offs, what's been proposed does (unlike my birdbath...) seem reasonable, but it also means we no longer have a predictable dataset, and it means DBAs and the Code Table Committee need to consider something other than what was earlier proposed before checking any boxes (or refusing to do so).

------------

Mostly unrelated, this (like all code tables) is widely available and the definition should not be polluted by our administrative stuff. I've been using https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html for that but don't really care about the location. (Maybe part of https://github.com/ArctosDB/arctos/issues/5326 if that comes to exist.)

**Comment by happiah-madson** [comment]
> It's been proposed to bring individual terms from other groups in, without blessing the entirety of the group. We like http://marineregions.org/mrgid/1902 (Atlantic Ocean, a General Sea Area) but not http://marineregions.org/mrgid/8665 (Off North West Amphipods, another General Sea Area). Can this be done in such a way that a future user won't point to the former while demanding the latter? (Or if we're OK with the latter, then can we just say that now?)

My apologies if this is self evident or if I am being obtuse, but is there a reason not to just opt for the latter? They both have MRGIDs and coordinates. Clearly we generally speaking have a preference for IHO, but as long as there is a Name Source, and a MRGID, does it matter?

**Comment by falco-rk** [comment]
What do you mean by: 
>Water Body Type is a marineregions.org PlaceType.

**Comment by dustymc** [comment]
>reason not to just opt for the latter? 

We spent a bunch of time trying to define this where a request could be auto-approved. There are things in marineregions which we clearly want (seas), and things which we clearly(ish-I-think...) do not (random '.. Amphipods' dots scattered all over the place). We're not quite sure how to define waterbody itself, but we're pretty sure my plastic birdbath or a [city](http://marineregions.org/mrgid/32512) in marineregions isn't one, so we're hoping that listing types in marineregions can act as a proxy - and a functional decision-maker for inclusion in Arctos. If we have to decide if individual things are more like the seas or the crazy dots then this becomes arbitrary and probably painful for everyone.

If we fail to exclude the random dots (we'd not preemptively add them, but I'd also not favor rejecting a request if we just expand the list to include oceans) it's near arbitrary but it's not OUR arbitrary, so maybe that is the way to go. (And I still don't care, other than trying to understand the shape of this concept and dodge future bullets!)

>mean by

The list following that clause is the important bit - `` in (IHO Sea Area)`` 

* allows  http://marineregions.org/mrgid/1906 because <img width="188" alt="Screenshot 2024-07-02 at 07 38 23" src="https://github.com/ArctosDB/arctos/assets/5720791/d5ede20e-51f9-45e0-879c-6fd3399c7d88">
* does not allow http://marineregions.org/mrgid/8665 because <img width="214" alt="Screenshot 2024-07-02 at 07 40 46" src="https://github.com/ArctosDB/arctos/assets/5720791/db267227-d1be-409b-972e-103586dd14ad"> isn't in the list
* is easily expandable to eg `` in (IHO Sea Area | Lake)`` if we want to add http://marineregions.org/mrgid/5659 (and I think "we" do, kinda immediately)

**Comment by happiah-madson** [comment]
ohhhh, is water body type the Arctos [code table field](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctwater_body)?

>Water Body Type is a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.

Does this mean that when you're making a new value in the code table (1) the water body type must be a marineregions.org PlaceType (which must be an IHO Sea Area, per Teresa's first bullet?) (2) water body is 'name' on marineregions.org and (3) documentation URL is the mrgid?

**Comment by dustymc** [comment]
>is

All correct.

<img width="1722" alt="Screenshot 2024-07-02 at 08 15 28" src="https://github.com/ArctosDB/arctos/assets/5720791/aec86fcd-acc9-4a41-a8fc-0e66f5caada7">

**Comment by genevieve-anderegg** [comment]
I like Teresa's definition, and think we can expand to Placetype = Lake as well, because we will be wanting those.

**Comment by dustymc** [comment]
>Teresa's definition

@genevieve-anderegg please clarify, are you advocating for entirely arbitrary data or am I very badly misunderstanding something?

**Comment by genevieve-anderegg** [comment]
Limiting by Placetype (acceptable terms being IHO Sea Area and Lakes) is a good idea. With the biggest sticking point being a lack of "Pacific Ocean" and "Atlantic Ocean" in Placetype = IHO Sea Area, maybe we just accept that and when we have a specimen with "Pacific Ocean" we just do two attributes for the North and South Pacific?
(Also, when looking on marineregions at the previews for shapes of the Pacific, they all seem to get cut off at the antemeridian in the middle of the ocean, so maybe we don't want to use any of them anyway because they don't fit we think of as the whole Pacific Ocean? https://www.marineregions.org/gazetteer.php?p=details&id=1908 )

I want this to move forward, so I am definitely open to changing my opinion to make that happen. I am ok with a more "administrative" definition for waterbody. 

@mkoo what do you think?

**Comment by Jegelewicz** [comment]
I don't know exactly when we made the decision that the terms in this attribute would be the exact same terms we already have in higher geography, but that is what the [current definition](https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2168430672) does.

As for making this term follow a source or sources without exception, I also don't know where that idea came into being, although the [definition that was given the thumbs up](https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2168430672) by only one person who is a code table manager does exactly that.

We are not following our own rules in this issue as the definition was changed after Nicole signed off on the term. A new definition was given the thumbs up by Michelle and Genna, but I am not 100% sure they realized they were just authorizing the terms currently in higher geography or that the expectation was that we would ONLY add terms from that "authority" and no others.

[DarwinCore waterBody](https://dwc.tdwg.org/terms/#dwc:waterBody) 

> Recommended best practice is to use a controlled vocabulary such as the Getty Thesaurus of Geographic Names.

GBIF does not DO anything with this term as far as I can tell, it is just a string and they don't try to match it to anything as they do geography or taxon names so we aren't trying to meet some community standard by sticking to a single authority.

There is nobody saying that any authority is any good, but I think a lot of people just want consistency. I do not believe we are going to find one or two authorities that will satisfy all the needs of all the collections in Arctos and I don't see any reason for us to try to BE the authority either. 

I am confused as to why we can't add terms that aren't in the authority we are currently using as higher geography (also the duplication of effort seem strange).

> Also, when looking on marineregions at the previews for shapes of the Pacific, they all seem to get cut off at the antemeridian in the middle of the ocean, so maybe we don't want to use any of them anyway because they don't fit we think of as the whole Pacific Ocean? https://www.marineregions.org/gazetteer.php?p=details&id=1908 )

Why do we care what the shapefile looks like? We have already said that isn't part of this term?

IMO this has become a bit ridiculous. We have things collected in water and the broader community (TDWG, GBIF) have specified where the name for the water should go and we can't seem to make that happen. Why is this so terribly difficult?

**Comment by dustymc** [comment]
>terms in this attribute would be the exact same terms we already have in higher geography,

I might have mentioned that a few times....

> aren't trying to meet some community standard

So why not free-text, or define this as "unique strings" if we want a sort of dictionary? If it's just strings then let's don't make it complicated! (And there's the fundamental problem, I am still uncertain as to what we're trying to DO, hence all the struggling with the definition.)

> (TDWG, GBIF) have specified where the name for the water should go 

So close this, use existing (or request new Sources of) spatial geography, and do something with https://github.com/ArctosDB/arctos/issues/7348?

**Comment by Jegelewicz** [comment]
Strings will never be CONSISTENT. A pick list at least provides a spell check.

> o close this, use existing (or request new Sources of) spatial geography, and do something with https://github.com/ArctosDB/arctos/issues/7348?

??? it's supposed to go in waterBody - that's what we ARE asking for in this issue! It doesn't require spatial anything for anyone at this point. We are the ones who seem fixated on spatial things that we can't even use.

**Comment by Jegelewicz** [comment]
And yes - this will need to be added to the DwC mapping once we actually have it.

**Comment by dustymc** [comment]
>Strings will never be CONSISTENT.

<img width="379" alt="Screenshot 2024-07-03 at 15 24 27" src="https://github.com/ArctosDB/arctos/assets/5720791/5212eeee-5036-4ac5-90af-568d504c2062">



.


>upposed to go in waterBody - that's what we ARE asking for in this issue

At least one of us is very confused. I can map just about anything to waterbody, and we have [lots of things](https://arctos.database.museum/place.cfm?sch=geog&sea=_) that would fit very nicely.

>spatial things that we can't even use

??????????????????????????????

**Comment by genevieve-anderegg** [comment]
Sorry for bringing up that shape in marine regions, it peaked my interest more than anything. 

Agree with Teresa's comments above. We want a pick list of waterbodies that do NOT have shapes associated (but could maybe one day at a later date, wayy down the road) that will go into the DwC waterbody field. We are just arguing over which authorities to use as sources because we don't want it filled with junk. 
Why can't we just outline our source, and then say if someone requests something else then the committee will discuss it? I think the Placetypes of IHO Sea Areas and Lakes in marineregions is expansive enough for now. If someone wants something else (like a value that is another kind of PlaceType) then they can request it and we can discuss. I know we don't want junk; if we start with these values we FOR SURE want, and then have a good system in place for requests and discussion, then that should be just fine. We meet every month anyway. And if we ever need to expand our definition, then we will have a conversation!!

I like Teresa's definition from above (with minor edits): 

Controlled vocabulary for bodies of water available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

- The Body of water is listed in in https://www.marineregions.org/ with "PlaceType" in (IHO Sea Area, Lakes)
- Documentation URL is a marineregions.org MRGID.
- Water Body Type is a marineregions.org PlaceType. For example, Water Body=Arctic Ocean; Water Body Type=IHO Sea Area; Documentation URL=http://marineregions.org/mrgid/1906.

If the body of water does not meet those criteria, the documentation should include a link to the resource that describes the term and the description should include a short description of the body of water. The Geography committee will discuss proposed waterbodies that do not meet the criteria.

**Comment by dustymc** [comment]
>then the committee will discuss it

https://github.com/ArctosDB/arctos/issues/7374#issuecomment-2201439792

I am ***begging*** for ANYTHING other than arbitrary, at the whims of whoever shows up to whatever thing on whatever day in whatever mood. An incoming collection (or someone who wants to upgrade, etc., etc.,) should be able to know whether this is appropriate for their data or not. I still don't care what the criteria for inclusion are, but I do think we need some. (https://en.wikipedia.org/wiki/Lists_of_bodies_of_water is cool...)

Not following something spatial-authority-ish would likely prevent making this spatial, but I think I've heard that would not be entirely desirable anyway.

**Comment by mkoo** [comment]
This is a topic in both CT and Geography committees this week. This is the definition from the Geog committee:

A body of water on the planet Earth forming a physiographical feature; for example, a sea within a ocean basin, or a lake or reservoir on a continent as listed in Marine Regions (marineregions.org) and/or Wikipedia. Acceptable values are listed on the Wikipedia list of Waterbodies (https://en.wikipedia.org/wiki/Lists_of_bodies_of_water) and associated linked lists therein, or in the IHO Sea Areas (https://marineregions.org/sources.php; https://www.vliz.be/en/imis?dasid=5444&doiid=323)

This lays out the sources for acceptable terms. CT committee will update the table definition once reviewed and approved. Then these are the tasks:

- [ ] import currently used IHO values into table water_body (or I can send a list -- let me know what's easier)
- [ ] add to  DwC IPT mapping

**Comment by dustymc** [comment]
>definition

Is there something in  (https://marineregions.org/sources.php; https://www.vliz.be/en/imis?dasid=5444&doiid=323) that's not in https://en.wikipedia.org/wiki/Lists_of_bodies_of_water? I'm fine with either, but if we can simplify I suggest we should.

>import

I would very strongly prefer individual requests. Definitions are needed, etc., ironing out that up front, before some mess turns into data, is about always the easiest solution. Let me know if that's not possible and I'll do something....

> DwC IPT 

Strongly prefer that be addressed via https://github.com/ArctosDB/arctos/issues/7348 (which seems to be scattered and abandoned - maybe there's no interest in a broad solution and we should instead kill that and related issues??)

**Comment by genevieve-anderegg** [comment]
Any update on this? 
Michelle and the committee decided to start with importing current IHO values (https://arctos.database.museum/place.cfm?sch=geog, geography remark contains "iho"), and then use individual requests for additions after.

**Comment by dustymc** [comment]
>update

See above, I have questions.

> current IHO values

See also above. (I'll open Issues if nobody else is willing, I could still use some help or I'll probably start wherever you don't have data.)

**Comment by genevieve-anderegg** [comment]
> Is there something in (https://marineregions.org/sources.php; https://www.vliz.be/en/imis?dasid=5444&doiid=323) that's not in https://en.wikipedia.org/wiki/Lists_of_bodies_of_water? I'm fine with either, but if we can simplify I suggest we should.

Perhaps, but that is the definition the committee decided upon and we all support, so I don't feel like further simplification is necessary.

 

> import currently used IHO values into table water_body (or I can send a list -- let me know what's easier)

[IHO_Geography_for_waterBody.csv](https://github.com/user-attachments/files/16680613/IHO_Geography_for_waterBody.csv)
Is this what you need? I can fill out the table in the format we desire for the waterbody table and make an issue
